tests/actions/buildiso/append_line_test.py

```diff
@@ -77,3 +98,38 @@ def test_generate_profile_install(request, cobbler_api, create_distro, create_pr
         % (originalname, originalname)
     )
 
+
+def test_generate_profile_rhel7(cobbler_api, create_distro, create_profile):
+    # Arrange
+    test_distro = create_distro()
+    test_distro.breed = "redhat"
+    cobbler_api.add_distro(test_distro)
+    test_profile = create_profile(test_distro.name)
+    blendered_data = utils.blender(cobbler_api, False, test_profile)
+    test_builder = AppendLineBuilder(test_distro.name, blendered_data)
+
+    # Act
+    result = test_builder.generate_profile("redhat", "rhel7")
+
+    # Assert
+    # Very basic test yes but this is the expected result atm
+    # TODO: Make tests more sophisticated
+    assert result == f"  APPEND initrd=/{test_distro.name}.img inst.ks=default.ks"
```

Comment:
Same as above; additionally, we modify upstream's `append` to ` APPEND` in these tests